### ISSUE TITLE
Invalid peers

### ISSUE THREAD

**Issue by altanozlu** [scylladb/scylla-rust-driver#235]
In gocql when there is a glitched node it'll log and skip the node.
example log:
`Found invalid peer '[HostInfo hostname="" connectAddress="x.x.x.x" peer="y.y.y.y" rpc_address="<nil>" broadcast_address="<nil>" preferred_ip="x.x.x.x" connect_addr="x.x.x.x" connect_addr_source="connect_address" port=9042 data_centre="" rack="" host_id="00000000-0000-0000-0000-000000000000" version="v0.0.0" state=UP num_tokens=0]' Likely due to a gossip or snitch issue, this host will be ignored`
But in here it's missing and `session.await_schema_agreement().await?;` will raise an error like: `Error: Protocol Error: Row is not uuid type as it should be`
Should we skip in here too ? @psarna

**Comment by psarna** [comment]
Yes, logging a warning and skipping sounds reasonable, we definitely shouldn't get stuck on an error.
